src/pages/diff.tsx

```diff
@@ -46,6 +48,7 @@ const Diff = () => {
   const SectionHeader = ({ section }: { section: string }) => {
     if (section === 'metadata') section = 'Metadata';
     else if (section === 'precompiles') section = 'Precompiles and Predeploys';
+    else if (section === 'opcodes') section = 'Opcodes';
```

Comment:
To clean this up, on `main` you'll see I migrated to a `SECTION_MAP` const:
https://github.com/mds1/evm-diff/blob/25de1747b21137fbca148dee06d1f93152796ca9/src/pages/diff.tsx#L10-L14